lib/ui/viewer/file/file_caption_widget.dart

```diff
@@ -13,7 +13,7 @@ class FileCaptionWidget extends StatefulWidget {
 }
 
 class _FileCaptionWidgetState extends State<FileCaptionWidget> {
-  int maxLength = 280;
+  static const int maxLength = 5000;
```

Comment:
Also, let us start showing the counter text after 1000 characters (right now it seems to be after 100 characters, but with the new limit and 10 line height that's too less)